build/cqs.tsv

```diff
@@ -290,9 +289,9 @@ org.apache.hadoop:hadoop-yarn-common	2.8.5	provided
 org.apache.hbase:hbase-server	2.2.3	provided
 org.apache.kafka:kafka_2.11	2.1.1	provided
 org.apache.kafka:kafka-clients	2.1.1	provided
-org.apache.spark:spark-catalyst_2.11	2.3.1	provided
-org.apache.spark:spark-core_2.11	2.3.1	provided
-org.apache.spark:spark-sql_2.11	2.3.1	provided
+org.apache.spark:spark-catalyst_2.11	2.4.7	provided
+org.apache.spark:spark-core_2.11	2.4.7	provided
+org.apache.spark:spark-sql_2.11	2.4.7	provided
```

Comment:
Do we need to change the minimum supported Spark version?

Reply:
no, it should still work with 2.3.x (although i did not try it). there are no 2.3.x_2.12 spark artifacts iirc so i had to bump it.